pyro/ops/einsum/paths.py

```diff
@@ -15,7 +15,7 @@ def ssa_to_linear(ssa_path):
         >>> ssa_to_linear([(0, 3), (2, 4), (1, 5)])
         [(0, 3), (1, 2), (0, 1)]
     """
-    ids = np.arange(1 + max(map(max, ssa_path)), dtype=np.int32)
+    ids = torch.arange(1 + max(map(max, ssa_path)), dtype=torch.long)
```

Comment:
Please revert. This should be kept as close as possible to the upstream version.

Reply:
@fritzo sorry, I didn't know about that. Do you want me to revert in another pull request?

Reply:
no worries. would you mind opening a tiny PR that reverts this file and setup.py. The other changes are nice, thanks for cleaning up!

Reply:
Sorry, I should have checked with you on the change in paths.py. Thanks for reverting this part, @fehiepsi.